packaging/rpm/microshift.spec

```diff
@@ -175,9 +185,7 @@ fi
 %post selinux
 
 %selinux_modules_install -s %{selinuxtype} %{_datadir}/selinux/packages/%{selinuxtype}/microshift.pp.bz2
-if /usr/sbin/selinuxenabled ; then
-    %microshift_relabel_files
-fi;
+%microshift_relabel_files
```

Comment:
Hmm, didn't we have that macro before? where is it gone?

Weren't we supposed to only relabel if selinux is enabled?